src/Chapter1.hs

```diff
@@ -631,8 +646,11 @@ Implement a function that returns the sum of the last two digits of a number.
 Try to introduce variables in this task (either with let-in or where) to avoid
 specifying complex expressions.
 -}
-
-sumLast2 n = error "sumLast2: Not implemented!"
+sumLast2 :: Int -> Int 
+sumLast2 n = onesDigit n + tensDigit n
+  where
+    onesDigit p = abs p `mod` 10
+    tensDigit p = (abs p `div` 10) `mod` 10
```

Comment:
That is a wonderful solution! 👏🏼  You correctly noticed that it is the `div` and `mod`, cool 😎 

One hint to make your solution even shorter: you can see that you use both:

```haskell
mod (abs p) 10
div (abs p) 10
```
The standard library has the [`divMod`](https://hackage.haskell.org/package/base-4.14.0.0/docs/Prelude.html#v:divMod) function, that actually combines inside both `div` and `mod`. And this is exactly what you use!.

So you could write it this way:

```haskell
(first, second) = divMod (abs n) 10
```

You can see how we could pattern match on the pair 🙂